src/core/components/radio/index.tsx

```diff
@@ -129,6 +138,8 @@ interface RadioProps extends InputHTMLAttributes<HTMLInputElement>, Props {
 	defaultChecked?: boolean
 	label?: ReactNode
 	supporting?: ReactNode
+	error: boolean
+	optional?: boolean
```

Comment:
Optional is passed as a default, so doesn't need to be marked as optional... Urgh, confusing! To put it another way, I think the question mark is redundant here

Reply:
Actually, maybe not! It doesn't look like TypeScript acknowledges those default props. Perhaps we should move away from default props to use JavaScript default arguments